src/mixins/homeMetaInfo.js

```diff
@@ -0,0 +1,3 @@
+export default {
+  metaInfo() { return { title: 'Kodilan.com' }; },
```

Comment:
; should not be, because it is a single line.

Is there a reason not to do the arrow function?
`metaInfo () => { title: 'Kodilan.com' },`

Reply:
I've tried to follow the general kodilan code base style. 
And I suppose the way you presented might be syntacticly invalid.